AISKU/src/AISku.ts

```diff
@@ -103,12 +103,27 @@ function _chkDiagLevel(value: number) {
     return value && value > 0;
 }
 
+function _parseCs(config: IConfiguration & IConfig, configCs: string | IPromise<string>) {
```

Comment:
Moved this out of the class as it's basically invariant so it doesn't need access to any of the properties in the closure.